app/package.json

```diff
@@ -16,8 +16,7 @@
   "devDependencies": {
     "@babel/core": "^7.3.4",
     "@babel/plugin-proposal-class-properties": "^7.3.4",
-    "@babel/preset-env": "^7.3.4",
-    "parcel-bundler": "^1.12.0"
+    "@babel/preset-env": "^7.3.4"
```

Comment:
We should keep parcel here and remove it from the root `package.json`.

Reply:
Fixed in https://github.com/aragon/aragon-react-boilerplate/pull/48

Reply:
Thanks Daniel!